### ISSUE TITLE
Searchkick: Use different analyzers in the same model

### ISSUE THREAD

**Issue by tschimbr** [eonum/drg-search#35]
I could not yet find out how to use different analyzers (stemmers, language dependent tokenizers..) within the same model in searchkick.

**Comment by tschimbr** [comment]
And ignore stop words if possible language dependent

**Comment by asittampalam** [comment]
https://github.com/ankane/searchkick#advanced-mapping - Using elasticsearch DSL within searchkick seems to be possible.

**Comment by tschimbr** [comment]
yes i've read this documentation. but the combination of searchkick features and advanced mappings seems to be somehow tricky. feel free to experiment.
